frontend/src/components/my-items/AddItemForm.jsx

```diff
@@ -30,6 +30,7 @@ export default function AddItemForm (props) {
     formData.append('description', description)
     formData.append('status', switchIsAvailable ? STATUS.AVAILABLE : STATUS.NOT_AVAILABLE)
     formData.append('image', fileInput.current.files[0])
+    formData.append('numberOfTimesBorrowed', 0)
```

Comment:
I don't think you need to set this here, instead you can set a default in the Item model

ie
  numberOfTimesBorrowed: {
    type: Number,
    default: 0
  },